source/dbo/Stored Procedures/Power BI Reporting/uspGenerateMultiSystemTransfers.sql

```diff
@@ -0,0 +1,52 @@
+﻿CREATE PROCEDURE [dbo].[uspGenerateMultiSystemTransfers]
+AS
+BEGIN TRY
+	TRUNCATE TABLE [dbo].[MultiSystemTransfers]
+
+	-- transfers in NTBS to regions not yet on NTBS
+	INSERT INTO [dbo].[MultiSystemTransfers] ([NotificationID],
+		[NotificationDate],
+		[ETSID],
+		[Requester],
+		[RequestedOrganisation],
+		[CreatedDate])
```

Comment:
Worth having a field for transfer type here? So we can more easily distinguish the two types we're interested in:
* NTBS transfer attempt to ETS region
* ETS transfer attempt to NTBS region

Reply:
I think this is a good idea :-)

Reply:
Added a source system column